src/member/apps.py

```diff
@@ -3,3 +3,9 @@
 
 class MemberConfig(AppConfig):
     name = 'member'
+
+    def ready(self):
+        # import signal handlers
+        # (every models imported inside handlers will be instantiated as
+        # soon as the registry is fully populated.)
+        import member.signals.handlers  # noqa
```

Comment:
@lingxiaoyang I understand your point, but now we have only one signal receiver, and that the receiver method have just two code's lines. But in the future maybe we can have many signal receivers with many code's lines. Maybe is inconvenient to keep all signal logic handled into the `ready` method.
I followed the suggestion explained in the 3rd cooment... http://stackoverflow.com/questions/2719038/where-should-signal-handlers-live-in-a-django-project 

what do you think?